feedback/models.py

```diff
@@ -1,12 +1,10 @@
-
-
 from django.db import models
 from django.contrib.auth.models import User
 from django_extensions.db.models import TimeStampedModel
 
 
 class Feedback(TimeStampedModel):
-    user = models.ForeignKey(User)
+    user = models.ForeignKey(User, on_delete=models.CASCADE)
```

Comment:
Can you explain why you chose models.CASCADE instead of models.SET_NULL? I don't know a lot about why one may be preferable over the other, and I hope that there isn't a case where we straight up delete a user (other than the ones that I do for testing) who may have submitted feedback, but I'd rather never lose feedback records but can be convinced if we have a good reason for CASCADE.

Reply:
After discussion with Sam we decided that since the field is not currently nullable we'd have to add that as well to use SET_NULL which is just outside of the scope of this issue and we're not going to worry about it. Just like...don't delete users, y'all.